dist/angular-breadcrumb.js

```diff
@@ -284,8 +284,95 @@ function BreadcrumbLastDirective($interpolate, $breadcrumb, $rootScope) {
 }
 BreadcrumbLastDirective.$inject = ['$interpolate', '$breadcrumb', '$rootScope'];
 
+function BreadcrumbTextDirective($interpolate, $breadcrumb, $rootScope) {
+
+    return {
+        restrict: 'A',
+        scope: {},
+        template: '{{ncyBreadcrumbLabel}}',
+
+        compile: function(cElement, cAttrs) {
+            // Override the default template if ncyBreadcrumbText has a value
+            var template = cElement.attr(cAttrs.$attr.ncyBreadcrumbText);
+            if(template) {
+                cElement.html(template);
+            }
+            
+            var separator = cElement.attr(cAttrs.$attr.ncyBreadcrumbTextSeparator);
+            if (!separator) {
+                separator = ' / ';
+            }
+
+            return {
+                post: function postLink(scope) {
+                    var labelWatchers = [];
+                    
+                    var registerWatchersText = function(labelWatcherArray, interpolationFunction, viewScope, step) {
+                        angular.forEach(getExpression(interpolationFunction), function(expression) {
+                            var watcher = viewScope.$watch(expression, function() {
+                                step.ncyBreadcrumbLabel = interpolationFunction(viewScope);
+                                renderLabel();
+                            });
```

Comment:
We build the entire chain of states in the `renderLabel`, so specific step data is not used. L313 is not needed and the whole watch could be reduced to:
```javascript
var watcher = viewScope.$watch(expression, renderLabel);
```